lib/phoenix_live_view/html_tokenizer.ex

```diff
@@ -191,24 +247,25 @@ defmodule Phoenix.LiveView.HTMLTokenizer do
 
   defp handle_comment("-->" <> rest, line, column, buffer, acc, state) do
     state = update_in(state.context, &[:comment_end | &1])
-    handle_text(rest, line, column + 3, ["-->" | buffer], acc, state)
+    # TODO: should we receive the offset here too?
+    handle_text(rest, {line, 0}, column + 3, ["-->" | buffer], acc, state)
```

Comment:
I guess we need to also get the offset here to keep it consistent

Reply:
YEs!